frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -29,9 +30,9 @@ const StyledTitle = styled(Card)`
 `
 const HorizontalContent = styled.div`
     display: grid;
-    grid-template-columns: auto auto auto;
+    grid-template-columns: auto auto auto auto;
     align-items: end;
```

Comment:
Remove ```align-items: end; ``` to get the headers (status, completed tasks, robot) to align better. Note that the battery will be centered so might have to add a header for battery as well or just a empty component instead of a header ```<></>```. And then wrapped in a similar style as the statusDisplay.